MAINTAINERS.yml

```
@@ -0,0 +1,1025 @@
+    status: maintained
+    maintainers:
+        - rveerama1
+    files:
+        - subsys/net/lib/coap/
+        - samples/net/sockets/coap_*/
+        - tests/net/lib/coap/
+
+"Networking: LWM2M":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - mike-scott
+    files:
+        - samples/net/lwm2m_client/
+        - subsys/net/lib/lwm2m/
+    labels:
+        - "area: LWM2M"
+
+"Networking: MQTT":
+    # Provisional entry, to be corrected
+    maintainers:
+        - rlubos
+    collaborators:
+        - jukkar
+    files:
+        - subsys/net/lib/mqtt/
+        - tests/net/lib/mqtt_packet/
+        - samples/net/mqtt_publisher/
+
+"NIOS-2 arch":
+    # Provisional entry, to be corrected
+    # TODO: The entries in the maintainers spreadsheet do not match CODEWONERS at all!
+    status: maintained
+    maintainers:
+        - ruuddw
+    collaborators:
+        - vonhust
+    files:
+        - arch/nios2/
+        - include/arch/nios2/
+    labels:
+        - "area: NIOS2"
+
+"nRF52 BSIM":
+    status: maintained
+    maintainers:
+        - aescolar
+    files:
+        - boards/posix/nrf52_bsim/
+    labels:
+        - "platform: nrf52_bsim"
+
+"POSIX API layer":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - pfalcon
+    files:
+        - include/posix/
+        - lib/posix/
+        - tests/posix/
+    labels:
+        - "area: POSIX"
+
+"Power management":
+    # TSC : Note this entry differs from the spreadsheet:
+    # pizi-nordic is not anymore working in Zephyr, therefore he cannot be a
+    # maintainer, and the are is orphaned until a new maintainer is appointed
+    status: orphaned
+    collaborators:
+        - wentongwu
+    files:
+        - include/power/power.h
+        - samples/subsys/power/
+        - subsys/power/
+    labels:
+        - "area: Power Management"
+
+"RISCV arch":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - nategraff-sifive
+    collaborators:
+        - mgielda
+    files:
+        - arch/riscv/
+        - include/arch/riscv/
+    labels:
+        - "area: RISCV"
+
+Sanitycheck:
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - nashif
+    files:
+        - scripts/sanitycheck
+        - scripts/sanity_chk/
+    labels:
+        - "area: Sanitycheck"
+
+Settings:
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - nvlsianpu
+    files:
+        - include/settings/
+        - subsys/settings/
+        - tests/subsys/settings/
+    labels:
+        - "area: Settings"
+
+Shell:
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - jakub-uC
+    files:
+        - include/shell/
+        - samples/subsys/shell/
+        - subsys/shell/
+        - tests/shell/
+        - tests/subsys/shell/
+    labels:
+        - "area: Shell"
+
```

```suggestion
Shields:
    status: maintained
    maintainers:
        - erwango
    collaborators:
        #To be confirmed
    files:
        - boards/shields
    labels:
        - "area: Shields"
```

@avisconti, @jfischer-phytec-iot would you be interested to be added as collaborators for `boards/shields`?

---

> @avisconti, @jfischer-phytec-iot would you be interested to be added as collaborators for `boards/shields`?

yes sure!

---

yes

---

Thanks guys, updated now.

---

@mike-scott, I forgot you on this. Would you be interested to be added as collaborator for `board/shields`?

---

Unresolve, waiting for @mike-scott feedback